codice/ddf
# DDF-4557 Simplify empty queries window

#### What does this PR do?
Simplify empty queries window

Also fixes a small css issue with map-settings.

#### Who is reviewing it? 
@andrewzimmer 
@GabrielFabian

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@djblue

#### How should this be tested?
Create and delete queries and make sure the view shows up when there are 0 queries.

#### What are the relevant tickets?
For GH Issues:
Fixes: #4557

#### Screenshots

##### Query-Selector Update
Before
<img src="https://user-images.githubusercontent.com/7236299/54464354-9332e080-4733-11e9-8d0d-3e6c8713faa9.png" height="300px" />
After
<img src="https://user-images.githubusercontent.com/7236299/54464688-ef4a3480-4734-11e9-95b8-a49de023ba82.png" height="300px" />

##### Map Settings Fix
Before
<img src="https://user-images.githubusercontent.com/7236299/54464548-5ca99580-4734-11e9-8212-bd9da3f27130.png" height="300px" />

After
<img alt="123" src="https://user-images.githubusercontent.com/7236299/54464616-95496f00-4734-11e9-84d8-a4b4d9923605.png" height="300px" />



#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
